packages/edit-site/src/store/actions.js

```diff
@@ -285,7 +285,7 @@ export const setPage =
 						.getEntityRecords( 'postType', 'wp_template', {
 							per_page: -1,
 						} )
-				)?.find( ( { slug } ) => slug === currentTemplateSlug );
+				 )?.find( ( { slug } ) => slug === currentTemplateSlug );
```

Comment:
I think prettier is complaining about the whitespace here 😄

<img width="693" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/faf0c11f-73c8-4714-9eca-5ab292ac094f">

Reply:
Thanks. 👍🏻 I saw that, and just reverted all the selector/actions changes which weren't relevant - already pushed. 

For some reason my IDE formatted it and it passed the pre-commit 🤷🏻 🤷🏻🤷🏻🤷🏻🤷🏻🤷🏻🤷🏻